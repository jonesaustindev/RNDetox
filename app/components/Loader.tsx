import React from 'react';
import {ActivityIndicator} from 'react-native';

export function Loader() {
  return <ActivityIndicator size={'large'} color={'black'} />;
}
