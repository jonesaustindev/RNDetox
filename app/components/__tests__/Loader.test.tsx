import React from 'react';
import {render} from '@testing-library/react-native';
import {Loader} from '../Loader';

it('renders loader', () => {
  const loader = render(<Loader />);
  expect(loader).toMatchSnapshot();
});
