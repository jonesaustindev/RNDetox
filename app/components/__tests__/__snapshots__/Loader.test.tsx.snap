// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loader 1`] = `
<ActivityIndicator
  color="black"
  size="large"
/>
`;
